src/ActionList/Item.tsx

```diff
@@ -98,12 +104,19 @@ export function Item({
   text,
   description,
   descriptionVariant = 'inline',
+  selected,
   leadingVisual: LeadingVisual,
   variant = 'default',
   ...props
-}: Partial<ItemProps>): JSX.Element {
+}: Partial<ItemProps> & {item: ItemInput}): JSX.Element {
   return (
-    <StyledItem variant={variant} {...props}>
+    <StyledItem
+      tabIndex={-1}
+      variant={variant}
+      {...(!!selected === selected && {'aria-selected': selected})}
```

Comment:
What is `!!selected === selected` checking for?

Also, I tend to prefer ternary's in cases like this because I think it makes it easier to understand what the expression will evaluate too (e.g. `selected ? {...} : null` vs `selected && {...}`)

Reply:
> What is `!!selected === selected` checking for?

That line is attempting to add the `aria-selected` attribute if and only if `selected` is `true` or `false`, not `undefined`. It could be rewritten as `{...(typeof selected === "boolean" && {'aria-selected': selected})}`, but since React omits attributes with `undefined` values, `aria-selected={selected}` is a readable equivalent.

> Also, I tend to prefer ternary's in cases like this because I think it makes it easier to understand what the expression will evaluate too (e.g. `selected ? {...} : null` vs `selected && {...}`)

It’s unnecessary if we use `aria-selected={selected}` as noted above, but I can accept this as a general guideline. It’s more explicit, and perhaps that will make PRC’s code more readable as a result, which is a good goal. Out of curiosity, are you aware of a lint run we could use for this (would we want a lint rule?)

Reply:
> `aria-selected={selected}` is a readable equivalent.

Replaced in https://github.com/primer/components/pull/1135/commits/1fa39d4414a0bc317bb14e7dabdfd28f9a260224!